common/flatpak-dir.c

```diff
@@ -8553,6 +8566,39 @@ flatpak_dir_prune (FlatpakDir   *self,
 
 }
 
+gboolean
+flatpak_dir_update_summary (FlatpakDir   *self,
+                            GCancellable *cancellable,
+                            GError      **error)
+{
+  g_autoptr(GError) local_error = NULL;
+  g_auto(GLnxLockFile) lock = { 0, };
+
+  if (error == NULL)
+    error = &local_error;
```

Comment:
Why do you need `local_error` at all?

Reply:
It was copied from `flatpak_dir_prune` and I kept it on the basis that some parts of libostree require `error` to be non-NULL, but now I've removed it after testing `create-usb` with a NULL error pointer for this function.

Reply:
(Not your bug, but…) requiring a `GError**` parameter to be non-`NULL` is an anti-pattern which should be removed from flatpak. Making the caller have to think/care about whether they pass in an error argument is not something I ever want to have, or to sow the seeds of doubt about.

Reply:
Yeah, it's unfortunate. Search the codebase for "if error is NULL" to see what I'm referring to.

Reply:
Although I think there's another instance of that too